.github/workflows/release_policy.yml

```diff
@@ -0,0 +1,104 @@
+name: Release Policy
+
+on:
+  workflow_dispatch:
+    inputs:
+      owner:
+        description: "Owner"
+        required: true
+        type: string
+      repo:
+        description: "Repository"
+        required: true
+        type: string
+      tag:
+        description: "Tag to release"
+        required: true
+        type: string
+  repository_dispatch:
+    types: [release-policy]
+
+jobs:
+  release-policy:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - name: Checkout current repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
```

Comment:
could we annotate the version of the checksum in a comment like in other workflows?
(same for the other GHA checksums in the workflow)